core/tests/protractor_utils/ExplorationEditorMainTab.js

```diff
@@ -470,14 +473,17 @@ var ExplorationEditorMainTab = function() {
     var stateEditorTag = element(by.tagName('state-content-editor'));
     var stateContentEditor = stateEditorTag.element(
       by.css('.protractor-test-state-content-editor'));
-    browser.wait(until.presenceOf(stateContentEditor), 5000,
+    waitFor.visibilityOf(
```

Comment:
I was afraid the condition was too harsh since this is a major point of "flakiness" during the whole learnerDashboard's waitForAngular issue, thus the downgrade to presenceOf instead of visibilityOf. 

Let's see if setContent evolves into a problem once this PR goes in!

Reply:
I think this is looking fine.